qiling/profiles/uefi.ql

```diff
@@ -1,52 +1,37 @@
 [OS64]
-heap_address = 0x500000000
-heap_size = 0x5000000
-stack_address = 0x7ffffffde000
-stack_size = 0x40000
-image_address  = 0x400000
-dll_address = 0x7ffff0000000
-entry_point = 0x140000000
-
+heap_address	= 0x78000000
+heap_size		= 0x02000000
+stack_address	= 0x77800000
+stack_size		= 0x00800000
+image_address 	= 0x77000000
+# entry_point	= 0x77400000
+# dll_address	= 0x7ffff000
 
 [OS32]
-heap_address = 0x5000000
-heap_size = 0x5000000
-stack_address = 0xfffdd000
-stack_size = 0x21000
-image_address  = 0x400000
-dll_address  = 0x10000000
-entry_point = 0x40000  
-
+heap_address	= 0x78000000
+heap_size		= 0x02000000
+stack_address	= 0x77800000
+stack_size		= 0x00800000
+image_address 	= 0x77000000
+# entry_point	= 0x77400000
+# dll_address	= 0x7ffff000
 
 [HOB_LIST]
 # EFI_GLOBAL_VARIABLE
-guid = 7739f24c-93d7-11d4-9a3a-0090273fc14d
-data1 = 0x7739f24c
-data2 = 0x93d7
-data3 = 0x11d4
-data4 = [0x9a, 0x3a, 0x0, 0x90, 0x27, 0x3f, 0xc1, 0x4d]
-vendortable = 0xffffffffffffffff
+Guid = 7739f24c-93d7-11d4-9a3a-0090273fc14d
+Table = 0x00000000ffffffff
 
 [DXE_SERVICE_TABLE]
-guid = 05ad34ba-6f02-4214-952e-4da0398e2bb9
-data1 = 0x05AD34BA
-data2 = 0x6F02
-data3 = 0x4214
-data4 = [0x95, 0x2E, 0x4D, 0xA0, 0x39, 0x8E, 0x2B, 0xB9]
+Guid = 05ad34ba-6f02-4214-952e-4da0398e2bb9
```

Comment:
why Guid, Table and not guid, table ?

Reply:
The code convention in EDK for names is to start with a capital letter. Since these entries replace specific UEFI variables with these names, they were changed to fit the names of the original vars they replace.